modules/pel/peltool/peltool.py

```diff
@@ -541,11 +541,30 @@ def printPELCount(path: str, config: Config, extension: str):
     print("{\n    \"Number of PELs found\": "+str(count)+"\n}")
 
 
+class CustomFormatter(argparse.HelpFormatter):
+"""
+This class is to enhance the formatting of argparse help messages 
+for arguments that have choices and allow multiple values.
+"""
+    def _format_action(self, action):
+        # Override the default formatting for choices when nargs='+' is used
+        if action.choices and action.nargs == '+':
+            metavar = self._get_default_metavar_for_optional(action)
```

Comment:
Does it work if we define our own metavars? We have a checklist to add metavars for all options to make it understandable for the user.

Reply:
I checked using metavar, the output is not changing. As [here](https://github.com/jtamames/SqueezeMeta/blob/master/lib/checkm/customHelpFormatter.py#L82), the metavar would contain the action.dest value.